File: src/com/gmail/marcosav2010/communicator/packet/wrapper/PacketReadException.java
package com.gmail.marcosav2010.communicator.packet.wrapper;

import java.io.IOException;

public class PacketReadException extends IOException {
    
	private static final long serialVersionUID = 4737890216554456161L;

	public PacketReadException(Exception exception) {
        super(exception);
    }
	
	public PacketReadException(String msg) {
        super(msg);
    }
}

File: core/src/main/java/com/gmail/marcosav2010/communicator/packet/wrapper/exception/PacketReadException.kt
package com.gmail.marcosav2010.communicator.packet.wrapper.exception

import java.io.IOException

class PacketReadException : IOException {

    constructor(exception: Exception) : super(exception)
    constructor(msg: String) : super(msg)

    companion object {
        private const val serialVersionUID = 4737890216554456161L
    }
}